carma-messenger-core/cpp_message/src/MobilityPath_Message.cpp

```diff
@@ -370,7 +387,90 @@ namespace cpp_message
         size_t array_length=(ec.encoded + 7) / 8;
         std::vector<uint8_t> b_array(array_length);
         for(size_t i = 0; i < array_length; i++) b_array[i] = buffer[i];
-        
+
         return boost::optional<std::vector<uint8_t>>(b_array);
     }
+
+    std::string Mobility_Path::bin2hex(std::string digits)
+    {
+        ROS_WARN_STREAM("INPUT: "<< digits);
+       std::stringstream res;
+       res << std::hex << digits; 
+       unsigned n;
+       res >> n;
+       std::bitset<8> set(n);
+       return set.to_string();
+    }
+
+    std::string Mobility_Path::hex2bin(std::string hex_string)
```

Comment:
This function and the one above look like duplicates. Can we move them to a separate file to share the implementation? Or maybe add it to the carma-utils package.

Reply:
These can just be deleted now. Per the change related to the conversion